spec/Unit/mock-template-part.php

```diff
@@ -0,0 +1 @@
+<h1><?php echo $args['title']; ?></h1>
```

Comment:
Quick discussion question, if I may 😄  ...

```
<?php echo $args['title']; ?>
```

... over ...

```
<?= $args['title'] ?>
```

We have a more verbose usage or minimal PHP code usage. 
Should we use minimal code as we are injecting a variable in HTML? Or does verbose output make more sense?

Reply:
I prefer `<?= $args['title'] ?>` in templates. `echo` is just a habit.

Are you good for me to change this one? And any others in a different PR?